packages/components/src/components/field-warn/field-warn.jsx

```diff
@@ -0,0 +1,12 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const FieldWarn = ({ message, className }) => <p className={classNames('dc-field-warn', className)}>{message}</p>;
```

Comment:
Do we need a separate component for FieldWarn or could it be combined with FieldError into a Field.`Type` component? (the styling seems to be the same except for the color)

Reply:
I think we need to keep using `FieldWarn` for future use, especially for warning fields with custom styling.

Reply:
Imo they are related in the same way `ButtonLarge` and `ButtonPrimary` are. These are not separate components they are one Button component that can be modified with a type - as the base styling and use is the same. 

I think FieldWarn and FieldError are the same? E.g. since the base style is the same if you need to update the styling for Field we'll have to update it in two places FieldWarn and FieldError.

either:
`<Field type="warn" />`
`<Field type="error" />`

or 
`<Field.Warn />`
`<Field.Error />`

Reply:
Yea I got the point. I'll investigate more then and find a better solution to implement it.